### PR TITLE
End round victory!

### PR DESCRIPTION
**Pull request by crossedfall**
:cl: Crossedfall
fix: Corrects the victory condition for the ship and updates the victory text accordingly
spellcheck: Corrects a minor grammatical issue for the go home objective
spellcheck: Adjusts the grammar for the completed/fail states for missions at round end
rscdel: Removes station objectives at round end
/:cl:

[why]: # Because it's kind of lame that the ship always fails its objectives. It'd be nice to get some victories every now and again. (Also we're a ship not a station! Get those lousy station objective outta here!)

image

### CODE REVIEW

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -555,17 +555,21 @@ SUBSYSTEM_DEF(ticker)
 	to_chat(world, "<br><FONT size=3><B>The ship objectives were:</B></FONT>")
 	var/count = 1
 	var/redtext = 0
+	if(!locate(/datum/objective/ftl/gohome) in get_ship_objectives())
+		redtext = 1
```
Comment (Qustinnus) [inline code/controllers/subsystem/ticker.dm]: TRUE

Comment (crossedfall) [reply code/controllers/subsystem/ticker.dm]: Why not suggest the changes then ya nerd. Y'all are the ones that made it `=1` in the first place.

Comment (crossedfall) [reply code/controllers/subsystem/ticker.dm]: Resolved in https://github.com/FTL13/FTL13/pull/3654/commits/f62e48889d881f5d826b72a084f180178ef6b938

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -554,18 +554,22 @@ SUBSYSTEM_DEF(ticker)
...
+			redtext = TRUE
+		else if(objective.check_completion() && !objective.failed)
+			to_chat(world, "<br><b>Objective #[count]</b>: [objective.explanation_text] <span class='greenannounce'>Completed!</span>")
 		else
-			to_chat(world, "<br><b>Objective #[count]</b>: [objective.explanation_text] <span class='boldannounce'>Fail.</span>")
-			redtext = 1
+			to_chat(world, "<br><b>Objective #[count]</b>: [objective.explanation_text] <span class='boldannounce'>Failed.</span>")
 		count++
-	if(redtext)
+	if(!redtext)
```
Comment (Qustinnus) [inline code/controllers/subsystem/ticker.dm]: ??

Comment (crossedfall) [reply code/controllers/subsystem/ticker.dm]: Yeah, big oops. Resolved in https://github.com/FTL13/FTL13/pull/3654/commits/c0fcc0c5e1228f7dea7fc3332dbb3cc27f70f2aa

**Pull request merged**